### ISSUE TITLE
Compile error in latest version: A rest parameter must be of an array type

### ISSUE THREAD

**Issue by taher435** [KillerCodeMonkey/ngx-quill#468]
When using the latest version of `ngx-quill` we get compilation error. Downgrading to version `7.0.2` fixes it. Assuming that the introduction of `onEditorChanged` caused this issue. This is the error I get in console.

```
ERROR in node_modules/ngx-quill/src/quill-editor.component.d.ts(92,70): error TS2370: A rest parameter must be of an array type.
```

Framework versions
```
Node: 8.9.1
OS: darwin x64
Angular: 6.1.9
```

**Comment by KillerCodeMonkey** [comment]
Oh i See. You Need typescript >= 3 for this Feature

Taher Dhilawala <notifications@github.com> schrieb am Mo., 29. Juli 2019,
06:21:

**Comment by KillerCodeMonkey** [comment]
fixed in 7.1.2. Avoiding rest parameters and use named parameters
